1.3/main.d.ts

```diff
@@ -973,13 +973,18 @@ declare module Meteor {
   var isServer: boolean;
   var isProduction: boolean;
   var release: string;
-  var settings: {
+  /** Global props **/
+  
+  /** Settings **/
+  interface Settings {
```

Comment:
Can you please change https://github.com/meteor-typings/meteor/blob/master/1.2/packages/meteor.d.ts since this is where it's defined?